### PR TITLE
New version: TortoiseSVN.TortoiseSVN version 1.14.29465

### PR DESCRIPTION
**Pull request by proudust**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80932)

### CODE REVIEW

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://mirrors.gigenet.com/OSDN//storage/g/t/to/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.5.29465-x64-svn-1.14.2.msi
```
Comment (yao-msft) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml:17]: I think this one is failing the domain validation. This should probably be the osdn.net urls like others.

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Ft%2Fto%2Ftortoisesvn%2F1.14.5%2FApplication%2FTortoiseSVN-1.14.5.29465-x64-svn-1.14.2.msi
```
Comment (Trenly) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml:17]:
```suggestion
  InstallerUrl: https://osdn.dl.osdn.net/storage/g/t/to/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.5.29465-x64-svn-1.14.2.msi
```
Better to use non-redirected URLs

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  InstallerSha256: B04B8004B0048593868BF938CEC4A6D412113AC7B7135FDF51E55B3251B5863B
+  ProductCode: '{E6F481E7-EF30-4811-A73C-1379B2C94595}'
+  InstallerSwitches:
+    Upgrade: /norestart
+    Custom: /norestart
+- Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Ft%2Fto%2Ftortoisesvn%2F1.14.5%2FApplication%2FTortoiseSVN-1.14.5.29465-win32-svn-1.14.2.msi
```
Comment (Trenly) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml:25]:
```suggestion
  InstallerUrl: https://osdn.dl.osdn.net/storage/g/t/to/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.5.29465-win32-svn-1.14.2.msi
```

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  InstallerSha256: E9471C6011292719282E292983F89516A81606F4CE5E226503F10480FAE3DC6D
+  InstallerSwitches:
+    Upgrade: /norestart
+    Custom: /norestart
+- Architecture: arm64
+  InstallerType: msi
+  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Ft%2Fto%2Ftortoisesvn%2F1.14.5%2FApplication%2FTortoiseSVN-1.14.5.29465-ARM64-svn-1.14.2.msi
```
Comment (Trenly) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml:32]:
```suggestion
  InstallerUrl: https://free.nchc.org.tw/osdn//storage/g/t/to/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.5.29465-ARM64-svn-1.14.2.msi
```

**Review by Trenly — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by proudust** [comment]
@MagicAndre1981 #118363 Is this okay?

**Pull request merged**